apps/sla/models.py

```diff
@@ -2,18 +2,33 @@
 SLA policy model definitions
 """
 
+from functools import cached_property
+
+from django.contrib.postgres import fields
+from django.db import models
+
 from apps.workflows.models import Check
 from osidb.models import Affect, Flaw, PsUpdateStream, Tracker
 
 from .exceptions import SLAExecutionError
 from .time import add_business_days, add_days
 
 
-class SLA:
+class SLA(models.Model):
     """
-    SLA definition and computation class
+    SLA definition and computation model
     """
 
+    DURATION_TYPES = [
+        ("business days", "Business Days"),
+        ("calendar days", "Calendar Days"),
+    ]
+
+    START_CHOICES = [
+        ("earliest", "Earliest"),
+        ("latest", "Latest"),
+    ]
```

Comment:
Just a question, why not to utilize `models.TextChoices` in this case?

Reply:
You're right, `TextChoices` is more appropriate here